recipes-devtools/deepstream/deepstream-6.0_6.0.0-1.bb

```diff
@@ -47,14 +48,9 @@ do_install() {
         install -m 0644 "$f" ${D}${DEEPSTREAM_PATH}/lib/
     done
     ln -sf libnvds_msgconv.so.1.0.0 ${D}${DEEPSTREAM_PATH}/lib/libnvds_msgconv.so
-    # Requires Python 3.6
-    rm -f ${D}${DEEPSTREAM_PATH}/lib/pyds.so
+    ln -sf libnvds_msgconv_audio.so.1.0.0 ${D}${DEEPSTREAM_PATH}/lib/libnvds_msgconv_audio.so
 
-    for i in 1 2; do
-        path="${DEEPSTREAM_PATH}/lib/triton_backends/tensorflow$i"
-	install -d ${D}$path
-	cp --preserve=mode,timestamps,links --no-dereference ${S}$path/* ${D}$path/
-    done
+    install -m 0644 ${S}${DEEPSTREAM_PATH}/lib/cvcore_libs/* ${D}${DEEPSTREAM_PATH}/lib/
```

Comment:
The install target should also be ...`lib/cvcore_libs/` to match the original structure

Reply:
Done